config/rds/config.go

```diff
@@ -85,9 +87,16 @@ func Configure(p *config.Provider) {
 			if a, ok := attr["endpoint"].(string); ok {
 				conn["endpoint"] = []byte(a)
 			}
+			if a, ok := attr["address"].(string); ok {
+				conn["address"] = []byte(a)
+				conn["host"] = []byte(a)
+			}
 			if a, ok := attr["username"].(string); ok {
 				conn["username"] = []byte(a)
 			}
+
+			conn["port"] = []byte(fmt.Sprintf("%v", attr["port"]))
```

Comment:
As far as I understand, we assume that `port` field of attr map will be full. How about handling a case where this value is null? Like that:

```go
if a, ok := attr["port"]; ok {
				conn["port"] = []byte(fmt.Sprintf("%v", a))
			}
```

Reply:
👍

Reply:
comment taken